incubator/jenkins-operator/README.md

```diff
@@ -1,3 +1,4 @@
+# DEPRECATED: Please consider using https://github.com/jenkinsci/kubernetes-operator
```

Comment:
Also number 6 from [PROCESSES.md#deprecating-a-chart](https://github.com/helm/charts/blob/master/PROCESSES.md#deprecating-a-chart) please also add the same note in `NOTES.txt`:
> Update READMEs and NOTES.txt to note that the chart is deprecated

**Quick note:** it took me a bit to find install instructions due to no chart README, but the instructions here are clear: https://jenkinsci.github.io/kubernetes-operator/docs/installation/#using-helm-chart Do you think it makes sense to also add a link to the docs helm install instructions?